pywb/recorder/s3uploader.py

```diff
@@ -43,7 +43,7 @@ def s3_upload_file(filename: str, bucket: str=None, object_name=None):
 
 
 def validate_warc_filename(filename: str):
-    warc_pattern = r'collections\/[\w\-]+\/archive\/[\w\-]+\.warc\.gz'
+    warc_pattern = r'collections\/[\w\-]+\/archive\/[\w\-.]+\.warc\.gz'
```

Comment:
Obviously it'll work because it's now more permissive but the unescaped dot will accept anything at all now, probably not an issue with our usage. Replace with `\.`?

Reply:
Doesn't need escaping in `[]` (at least in Python):

```python
import re
re.match("[.]+", ".")
# <re.Match object; span=(0, 1), match='.'>
re.match("[.]+", "test")
# None
```

Reply:
I did not know that!